### PR TITLE
[aclshow]: Add -m option to display current mirror sessions

### PR DESCRIPTION
**Pull request by stcheng**
Mirror is tightly coupled with ACLs. Add the -m option to display
the current mirror sessions so that for easy display the current
mirror sessions created.

### CODE REVIEW

**Comment by stcheng** [comment]
@andriymoroz-mlnx

**Comment by lguohan** [comment]
:shipit:

File: scripts/aclshow [hunk 1]
```diff
@@ -16,6 +16,7 @@ optional arguments:
   -r RULES,  --rules RULES    action by specific rules list: Rule_1,Rule_2
   -t TABLES, --tables TABLES  action by specific tables list: Table_1,Table_2
   -d,  --details              display detailed ACL info
+  -m,  --mirrors              display all mirror sessions
```
Comment (lguohan) [inline scripts/aclshow]: it is not intuitive to show mirror session in aclshow.

Comment (stcheng) [inline scripts/aclshow]: have a separate script 'mirrorshow'?

**Comment by stcheng** [comment]
show mirror_session covers the functionality

**Pull request closed without merging**